.gitignore

```diff
@@ -16,7 +16,7 @@
 build/_vendor/pkg
 /*.a
 docs/readthedocs/build
-
+genesis.json
```

Comment:
Hmm. Is it OK to add `genesis.json` to `.gitignore`?

Reply:
the `.gitignore` will be cleaned prior to merging, i put it like that so that i dont have to make a new one every time

Reply:
> the `.gitignore` will be cleaned prior to merging, i put it like that so that i dont have to make a new one every time

The local file `.git/info/exclude` can be handy for this.